Feature/fixes dx29 dx30 dx33 dx34

W3stside opened the pull request:
## JIRA Ticket fixes

1. DX-29: _MenuWallet needs to accurately reflect ether price_ 
 --> in progress, dealing w/contract issues
2. DX-30: _change wording "available balance" to your current balance in the receiving tokens_
 --> done
3. DX-33: _consider submission button changing to another wording_
 --> done (up to changes though)
4. DX-34: _Disallow getting to Submit Sell Order (last screen) if no sell amount entered_
 --> done (may need more functionality such as letting user know why they cannot move forward)

W3stside committed:
changed TokenItem BUYing token text to 'Current Balance' as opposed to less clear Available Balance

W3stside committed:
removed unnecessary cmments in MenuWallet and fixed Story bug

W3stside committed:
passing down sellAmount to component to use in conditional for ButtonCTA in case no sell amount selected

W3stside committed:
added sellAmount as prop to check if there is an amount entered > 0 and fixed typings in both container and component

W3stside committed:
changed Text on ButtonCTA to better signify click action to users

Velenir commented on src/components/OrderPanel/index.tsx:
```diff
@@ -25,7 +26,7 @@ const OrderPanel: React.SFC<OrderPanelProps> = ({ sellToken, buyToken }) => (
     <AuctionSellingGetting />
     {/* TODO: replace onclick with some logic (maybe: "to" prop) */}
-    <ButtonCTA onClick={() => console.log('Continuing to wallet')} to="/wallet">
+    <ButtonCTA onClick={() => console.log('Continuing to wallet')} to={+sellAmount > 0 && '/wallet'}>
```
You know what would be nice for indication that we can't go further, `disabled={sellAmount>0}`. Then pass this disabled all the way through to `<a disabled>` and change its style with `a.class[disabled]`.

Velenir replied:
Also I'm not sure but when `<Link to={not location object or string}/>` there may be an error when you click it.

W3stside replied:
OK cool - good idea. Will dig into this.

Velenir commented on src/components/TokenItem/index.tsx:
```diff
@@ -21,7 +21,7 @@ const TokenItem: React.SFC<TokenItemProps> = ({ onClick, ...rest }) => {
       <i data-coin={code}></i>
       <big>{name}</big><code>{code}</code>
-      <small>{mod && 'AVAILABLE'} BALANCE:</small>
+      <small>{mod && (mod === 'sell' && 'AVAILABLE' || 'CURRENT')} BALANCE:</small>
```
Why not use a ternary?

W3stside replied:
no reason

W3stside committed:
added buttonCTA-disabled for CSS disabling ButtonCTA if conditional

W3stside committed:
added ternary for buttonCTA className and text based on sellAmount

W3stside commented:
@Velenir can you quickly take a look at my buttonCTA changes? Cheers!

W3stside committed:
changed onClick to prevent moving forward if no sellAmount entered + kept CSS blockers and display
1. Dmitry sharply pointed out that tabing and hitting enter w/pointer-events: none still worked

W3stside committed:
changed conditional to ternary

W3stside committed:
removed useless comment
